frontend/public/locales/en/common.json

```diff
@@ -41,6 +41,8 @@
     "help": "Help",
     "contribute-translations": "Contribute Translations",
     "report-an-issue": "Report an Issue",
+    "vcs_browser": "Browse the source code",
+    "contribute": "Contribute to the App",
```

Comment:
```suggestion
    "contribute": "Contribute to the application",
```
maybe?

Reply:
I actually think we would want these shorter and to match the others just `Contribute` would be better. And from reading the other PR, it sounded a bit more like it should be `How to contribute` if we want a long version.